doc/api/esm.md

```diff
@@ -348,26 +351,24 @@ changes:
 
 * `specifier` {string} The module specifier to resolve relative to the
   current module.
-* Returns: {string} The absolute (`file:`) URL string for the resolved module.
+* Returns: {string} The absolute URL string that the specifier would resolve to.
```

Comment:
Nit

```suggestion
* Returns: {URL['href']} The absolute URL string that the specifier would resolve to.
```

Reply:
````
Error: Unrecognized type: 'URL['href']'.
Please, edit the type or update 'file:///Users/duhamean/Documents/node/tools/doc/type-parser.mjs'.
````

That doesn't seem worth it, the return type _is_ a string.

Reply:
I think it wants `typeof URL['href']`.

Reply:
This is the markdown documentation, not a JSDoc comment, it's not typescript. It wants a type that's defined in `tools/doc/type-parser.mjs`.

Reply:
:(